### PR TITLE
Transceiver onPictureLoss

### PR DESCRIPTION
**Pull request by zhuker**
*Description of changes:*

Exposes onPictureLoss callback to public api
`onPictureLoss` is called for both PLI and FIR rtcp packets

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/source/PeerConnection/Rtp.c [hunk 1]
```diff
@@ -136,6 +136,23 @@ STATUS transceiverOnBandwidthEstimation(PRtcRtpTransceiver pRtcRtpTransceiver, U
...
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PKvsRtpTransceiver pKvsRtpTransceiver = (PKvsRtpTransceiver) pRtcRtpTransceiver;
+
+    CHK(pKvsRtpTransceiver != NULL && onPictureLoss != NULL, STATUS_NULL_ARG);
+
+    pKvsRtpTransceiver->onPictureLoss = onPictureLoss;
+    pKvsRtpTransceiver->onPictureLossCustomData = customData;
+
+    CleanUp:
```
Comment (MushMal) [inline src/source/PeerConnection/Rtp.c:150]: Minor: Remove spaces before the label

**Comment by disa6302** [comment]
LGTM

File: src/source/PeerConnection/Rtp.c (hunk 1 above)
Comment (zhuker) [reply src/source/PeerConnection/Rtp.c]: done

**Comment by zhuker** [comment]
@disa6302 i suggest merging this PR before https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/673

**Comment by disa6302** [comment]
> @disa6302 i suggest merging this PR before #673

That is the plan :)

**Pull request merged**